TerminalDocs/customize-settings/global-settings.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Windows Terminal Global Settings
+description: Learn how to customize the global settings within Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 05/19/2020
+ms.topic: how-to
+ms.service: terminal
+---
+
+# Global settings in the Windows Terminal
+
+The properties listed below affect the entire window, regardless of the profile settings. These should be placed at the root of your settings.json file.
+
+## Default profile
+
+This sets the default profile that opens by typing `ctrl+shift+t`, typing the key binding assigned to `newTab`, or clicking the '+' icon.
+
+**Property name:** `defaultProfile`
+
+**Necessity:** Required
+
+**Accepts:** GUID as a string
+
+**Default value:** PowerShell's GUID
+
+<br />
+
+___
+
+## Disable dynamic profiles
+
+This sets which dynamic profile generators are disabled, preventing them from adding their profiles to the list of profiles on startup. For information on dynamic profiles, visit the [Dynamic profiles page](./../dynamic-profiles.md).
+
+**Property name:** `disabledProfileSources`
+
+**Necessity:** Optional
+
+**Accepts:** `"Windows.Terminal.Wsl"`, `"Windows.Terminal.Azure"`, and/or `"Windows.Terminal.PowershellCore"` inside an array
+
+**Default value:** `[]`
+
+<br />
+
+___
+
+## Dark/Light theme
+
+:::row:::
```

Comment:
Do the first two h2's on this page need this markup as well?

Reply:
Nah, this is only needed if there's an image next to the content.